config/locales/de.helpers.yml

```diff
@@ -40,7 +40,7 @@ de:
     submit:
       create: "Erstelle %{model}"
       submit: "Speichere %{model}"
-      update: "Aktualisiere %{model}"
+      update: "Speichern"
```

Comment:
Creates inconsistency between `en.helpers.yml` and `de.helpers.yml`; One requires a model to be set, and one doesn't. Although current focus is on the German version, I guess it's still a good idea to keep existing files clean and up to date. 

Why don't you use the already existent text from `t'(helpers.links.save)` or change the line in all concerned locale files.

Reply:
Yup, I'll adapt the `de.helpers.yml` accordingly.
I decided against using `helpers.links.save` because then I'd have to pass this string manually on all edit pages for all models vs just changing the string which is automatically retrieved in Rail's convention-over-configuration manner when it sees an `<%= f.submit ... %>` on an edit page.

Reply:
Okay, I agree.